### DISCUSSION TITLE
When I uploaded the Godaddy site to Vercel, the email referring to that domain stopped working.

### DISCUSSION THREAD

**Discussion by lsegouras** [vercel/community#3668] [Help] [answered]
### Summary

I added all the registers given by Godaddy as follow, but didn't work:

1. TXT registers:
Nome: D1928112
Valor TXT: v=spf1 include:secureserver.net -all

2. CNAME Registers:
Host: email
Aponta
para:
email.secureserver.net

3. MX Registers:
Host: @
Prioridade: 0
Alvo: smtp.secureserver.ne

Host: @
Prioridade: 10
Alvo: mailstore1.secureserver.net

4. SRV Registers:
Porta: 443
Protocolo: _tcp
Nome: @
Serviço: _autodiscover
Prioridade: 100
Peso: 1
Alvo: autodiscover.secureserver.net

### Steps to Reproduce

braulio.saraiva@esgimpact.com.br - this email does´t work anymore since I uploaded the site in Vercel.

**Comment by amyegan** [comment, marked as answer]
Hey @lsegouras. It looks like you changed your nameservers to Vercel instead of GoDaddy. Visit https://vercel.com/dashboard/domains to open your Vercel domain management page. From there you can access the setting page for your domain to add or modify DNS records.
